src/sentry/rules/conditions/tagged_event.py

```diff
@@ -32,6 +35,17 @@ class MatchType(object):
 )
 
 
+def get_project_release_cache_key(project_id):
+    return u"project:{}:latest_release".format(project_id)
+
+
+@release_created.connect(weak=False)
```

Comment:
Probably worth doing this on `post_delete` as well. I'm also a bit concerned about `date_released` changing and us not busting the cache, but it might not be a huge deal.

Reply:
changed so that the cache is invalidated on pre_delete for Release and ReleaseProject. Might be a bit redundant now, perhaps we only need ReleaseProject, but I just wanted to be safe.